package.json

```diff
@@ -47,7 +47,7 @@
     "ts-node": "cross-env ts-node -r tsconfig-paths/register -O '{\\\"module\\\":\\\"commonjs\\\",\\\"esModuleInterop\\\":true}'",
     "openapi-generator": "openapi-generator generate -g typescript-fetch --additional-properties=typescriptThreePlus=true",
     "schemas": "node ./scripts/schemas.js",
-    "postinstall": "node -e 'process.exit(process.env.CI ? 0 : 1)' || node ./scripts/postinstall.js",
+    "postinstall": "node -e 'process.exit(process.env.CI ? 0 : 1)' || test -n \"$SKIP_POSTINSTALL\" || node ./scripts/postinstall.js",
```

Comment:
Does this have good windows support? Maybe add it to the node expression instead?

Reply:
True, will fix.